UploadAndAnnotateImages/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "salesforce-app",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Salesforce App",
+  "scripts": {
+    "lint": "eslint **/{aura,lwc}/**",
+    "test": "npm run test:unit",
+    "test:unit": "sfdx-lwc-jest",
+    "test:unit:watch": "sfdx-lwc-jest --watch",
+    "test:unit:debug": "sfdx-lwc-jest --debug",
+    "test:unit:coverage": "sfdx-lwc-jest --coverage",
+    "prettier": "prettier --write \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
+    "prettier:verify": "prettier --list-different \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
+    "postinstall": "husky install",
```

Comment:
This doesn't work with `npm install`, presumably because Husky is meant to install hooks into the `.git/` repo, but this package is not associated with a repository at its root.

```bash
> salesforce-app@1.0.0 postinstall
> husky install

.git can't be found (see https://git.io/Jc3F9)
npm ERR! code 1
npm ERR! path /<My Repo Dir>/LWC-Mobile-Samples/UploadAndAnnotateImages
npm ERR! command failed
npm ERR! command sh -c -- husky install

npm ERR! A complete log of this run can be found in:
npm ERR!     /<My Home Dir>/.npm/_logs/2023-01-25T20_20_19_284Z-debug-0.log
```

We should look at either look at updating the husky configuration to support the subfolder, similar to [this example](https://scottsauber.com/2021/06/01/using-husky-git-hooks-and-lint-staged-with-nested-folders/), or remove the Husky requirement altogether.

Reply:
Thanks! I followed the example you shared and it seems to work now.